Payment UseCase/PaymentTest.java

```diff
@@ -3,6 +3,7 @@
 import static org.junit.jupiter.api.Assertions.*;
 //Testing PaymentTest class using JUnit test framework
 
+
 public class PaymentTest {
 

```

Comment:
//to improve youe code you can add Inner class for payment storage that make asuccess of the payment process by always return true